relay-server/src/actors/project_cache.rs

```diff
@@ -793,6 +794,23 @@ impl Service for ProjectCacheService {
 
             // Channel for envelope buffering.
             let (buffer_tx, mut buffer_rx) = mpsc::unbounded_channel();
+            let buffer = if config.cache_persistent_buffer_enabled() {
+                match SqliteBufferService::from_path(config.cache_persistent_buffer_path()).await {
+                    Ok(buf) => buf.start(),
+
+                    // TODO: how to propagate this error to the entire rely and stop it?
+                    // Should this panic? But this will only kill the service and not the relay.
```

Comment:
I don't know about propagating the error to the entire relay, but I do think in case of an error it should panic and kill the service. We can postpone this decision for now since it's not critical yet.